### PR TITLE
[FEATURE] added iniciatives module

### PR DESCRIPTION
**Pull request by victorol1**
Solves basecamp's: https://3.basecamp.com/4186608/buckets/13990309/todos/2159929012
* Added decidim-iniciatives module gem
* Limited sprockets version
* Added wicked_pdf gem to avoid migration conflicts

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,8 +11,11 @@ gem 'decidim-age_action_authorization', path: 'decidim-age_action_authorization'
 gem 'decidim-census', path: 'decidim-census'
 gem 'decidim-consultations'
 gem 'decidim-diba_census_api', path: 'decidim-diba_census_api'
+gem 'decidim-initiatives'
 gem 'decidim-ldap', path: 'decidim-ldap'
 
+gem "sprockets", "~> 3.7", "< 4"
+gem 'wicked_pdf'
```
Comment (tramuntanal) [inline Gemfile]: For sprockets and wicked_pdf comment why are we adding them.
Also for sprockets explain when we can get rid of this declaration.

Comment (victorol1) [reply Gemfile]: Done, changes applied.

**Comment by victorol1** [comment]
@tramuntanal added comments, check it when you have a moment

**Comment by victorol1** [comment]
@tramuntanal Issues resolved, ready to merge

**Pull request merged**